cpp/src/barretenberg/benchmark/honk_bench/compare_honk_to_plonk_ultra.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# This script is used to compare the results of honk_bench between baseline (master) and
+# the branch from which the script is run. Simply check out the branch of interest, ensure 
+# it is up to date with local master, and run the script.
+
+echo -e '\nComparing Ultra Plonk/Honk benchmarks.'
+# Set some directories
+BASE_DIR="$HOME/barretenberg/cpp"
+BUILD_DIR="$BASE_DIR/build-bench"
+BENCH_RESULTS_DIR="$BASE_DIR/tmp_bench_results"
+BENCH_TOOLS_DIR="$BUILD_DIR/_deps/benchmark-src/tools"
+
+# Install requirements (numpy + scipy) for comparison script if necessary.
+# Note: By default, installation will occur in $HOME/.local/bin.
+pip3 install -r $BUILD_DIR/_deps/benchmark-src/requirements.txt
```

Comment:
Add --user so that it definitely installs in user directories?

Reply:
done